pom.xml

```diff
@@ -72,11 +80,11 @@
         <dependency>
             <groupId>javax.jcr</groupId>
             <artifactId>jcr</artifactId>
-            <version>2.0</version>
         </dependency>
         <dependency>
             <groupId>org.osgi</groupId>
             <artifactId>osgi.annotation</artifactId>
+            <version>6.0.1</version>
```

Comment:
Please use the annotation dependency defined in the parent's depMgmt.

Reply:
I was not able to find the correct annotation to set the version without this dep. Can you point me to some place where the correct one is used?

Reply:
Pretty extensively commented in https://github.com/apache/sling-parent/blob/0f5091f47716d3738efda90ec2b5e67c942a9226/pom.xml#L1199

Reply:
Thanks .. somehow overlooked the link. I also wrongly simply removed the dependency instead of using the new groupId and artifactId. Now it works fine. Will update my PR.